ce/api/geo.py

```diff
@@ -124,5 +125,14 @@ def polygonToMask(nc, fname, poly, variable):
             raise Exception("Unable to determine projection parameters for GDAL "
                             "dataset {}".format(dst_name))
 
-        mask = rasterize((poly,), out_shape=raster.shape, transform=raster.affine, all_touched=True)
```

Comment:
No, it's still possible to open a NetCDF dataset that doesn't have any projection information (resulting in the identity affine). For the purposes of this application, that's an error condition, because there would then be no way to compare the raster to the input polygon.